internal/service/ecs/task_definition.go

```diff
@@ -143,6 +143,22 @@ func ResourceTaskDefinition() *schema.Resource {
 				ValidateFunc: validation.StringInSlice(ecs.NetworkMode_Values(), false),
 			},
 
+			"platform_family": {
```

Comment:
I think this needs to follow the api and be a runtime_platform block containing these instead of having it flattened

Reply:
I placed it based on platform family which was root based. Then the docs were half updated with this seemingly pointless extra object and platform family duplication 🤮

Reply:
K I've done it based on the https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html
so its (lazily typed here)

runtime_platform {
  operating_system_family = "LINUX"
  cpu_architecture = "ARM64"
}

Just need to run the tests etc and I'll push that in